exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/unnest/UnnestRecordBatch.java

```diff
@@ -103,16 +109,19 @@ public void update() {
       final MaterializedField field = incoming.getSchema().getColumn(typedFieldId.getFieldIds()[0]);
 
       // Get column size of unnest column.
+
       RecordBatchSizer.ColumnSize columnSize = getRecordBatchSizer().getColumn(field.getName());
 
+      RecordBatchSizer.ColumnSize rowIdColumnSize = getRecordBatchSizer().new ColumnSize(rowIdVector, null);
```

Comment:
Since we know `RowId` is always going to be an `IntVector`, we can directly use `TypeHelper.getSize()` instead of going through `RecordBatchSizer`.

Reply:
Good catch. Thanks.